pkg/url/types.go

```diff
@@ -31,6 +40,12 @@ type URLList struct {
 	Items           []URL `json:"items"`
 }
 
+type IngressURLList struct {
```

Comment:
we should be using the same struct as URL list

Reply:
Same as: https://github.com/openshift/odo/pull/3160/files#r425184472

Reply:
same as https://github.com/openshift/odo/pull/3160#discussion_r425576551